File: 프로그래머스/lv1/12935. 제일 작은 수 제거하기/제일 작은 수 제거하기.py
def solution(arr):
    answer = []
    m=arr[0]
    for i in range(1,len(arr)):
        if arr[i] < m:
            m=arr[i]
    for a in arr:
        if a != m:
            answer.append(a)
    if not answer:
        answer.append(-1)
    return answer

File: 프로그래머스/1/12935. 제일 작은 수 제거하기/제일 작은 수 제거하기.java
import java.util.*;

class Solution {
    public int[] solution(int[] arr) {
        if(arr.length<=1){
            return new int[]{-1};
        }
    
        int minValue = Arrays.stream(arr).min().orElseThrow();
        int[] answer = Arrays.stream(arr).filter(i->i!=minValue).toArray();
        return answer;
    }
}